beacon-node-oapi.yaml

```diff
@@ -424,7 +424,9 @@ components:
 
   headers:
     Eth-Consensus-Version:
-      description: Required in response so client can deserialize returned json or ssz data more effectively.
+      description: |
+        The active consensus version to which the data belongs. Required in response so client can deserialize returned json or ssz data 
+        more effectively.
```

Comment:
```suggestion
      description: The active consensus version to which the data belongs. Required in request and response so client or server can deserialize json or ssz data more effectively. When json is sent, this header matches the `version` field.
```

Reply:
maybe we can refer this in parameters like:

```
parameters:
  - $ref: '../../../beacon-node-oapi.yaml#/components/headers/Eth-Consensus-Version'
```

or if we want to have 2 different descriptions for outbound and inbound, we can have 2 different components header and reference them accordingly, so we can avoid duplications.

Reply:
@tbenr As discussed, it's not possible to directly refer to the `beacon-node-oapi.yaml#/components/headers/Eth-Consensus-Version` when the header is a parameter for POST endpoint.
And regarding the header description, adding a mention to the version property might be confusing since that might imply its presence and that's not the case for the POST request payloads for example and it could not be necessarily present in the GET endpoint responses.
So for now I removed the ", if using SSZ encoding." bit and kept the last wording

Reply:
uncommitted suggestion - can we resolve that?